### ISSUE TITLE
NPE when submitting edited POI

### ISSUE THREAD

**Issue by exploide** [osmandapp/Osmand#3746]
I used the OsmAnd editing plugin to make changes to a POI. After doing so, I hit submit and the app crashed. This happened multiple times. In the end, it worked by submitting them via the my places menu.

```
Version  OsmAnd~ 2.6.2
06.05.2017 5:00:26
Apk Version : 2.6.2 262
Exception occured in thread Thread[main,5,main] : 
java.lang.NullPointerException: Attempt to invoke interface method 'void net.osmand.plus.osmedit.dialogs.SendPoiDialogFragment$ProgressDialogPoiUploader.showProgressDialog(net.osmand.plus.osmedit.OsmPoint[], boolean, boolean)' on a null object reference
	at net.osmand.plus.osmedit.dialogs.SendPoiDialogFragment$2.onClick(SendPoiDialogFragment.java:108)
	at android.support.v7.app.AlertController$ButtonHandler.handleMessage(AlertController.java:157)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:241)
	at android.app.ActivityThread.main(ActivityThread.java:6217)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

**Comment by dmpr0** [comment]
Hi, we try to reproduce it, on different devices, but all work fine, can you please specify your Android version?

**Comment by exploide** [comment]
My Android version is 7.0 on a Sony Xperia X Compact.

(I haven't tried to reproduce this issue since I filled the bug, but on that day it occurred multiple times.)

**Comment by vshcherb** [comment]
Could you please specify where do you hit "submit" button?  Is it in the context menu?

**Comment by exploide** [comment]
I was able to reproduce the issue again on first try and took screenshots (language is German but I guess you recognize the menu).

I touched a POI, the info menu appeared, I selected the edit POI option and made changes. Once finished, I touched the save ("Speichern") button.
![screenshot_20170528-155515](https://cloud.githubusercontent.com/assets/5670236/26529443/208d0e66-43c0-11e7-855d-09bbded32de7.png)
That worked, the edit activity closed and I was back on the map. To submit my changes I touched upload ("Hochladen", that's the thing I called submit).
![screenshot_20170528-155534](https://cloud.githubusercontent.com/assets/5670236/26529445/24703350-43c0-11e7-9620-ac84104e4605.png)
And than, the App crashed.
